ui/shared/components/panel/variants/Variants.jsx

```diff
@@ -111,7 +112,11 @@ const Variant = React.memo(({ variant, isCompoundHet, mainGeneId, linkToSavedVar
         {isCompoundHet && <VariantIndividuals variant={variant} isCompoundHet />}
         {isCompoundHet && showReads}
       </Grid.Column>
-      <Grid.Column><Annotations variant={variant} /></Grid.Column>
+      <Grid.Column>
+        <Annotations variant={variant} />
+        <VerticalSpacer height={12} />
+        {variant.variantGuid !== undefined && <AcmgModal variant={variant} />}
```

Comment:
theres no need to check for undefined, any truthy value here is fine (i.e. just do `{variant.variantGuid && <AcmgModal variant={variant} />}`)